src/components/Spinner/Spinner.styles.ts

```diff
@@ -0,0 +1,45 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { addAlpha, getTheme } from '../../util/colors';
+import { mergeStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+import { SpinnerColor, SpinnerSize } from './types';
+
+export interface SpinnerClassNames {
+  root: string;
+  label: string;
+  circle: string;
+}
+
+const labelMargins = {
+  [SpinnerSize.XSMALL]: '-1px 0 0 6px',
+  [SpinnerSize.SMALL]: '-1px 0 0 8px',
+  [SpinnerSize.MEDIUM]: '-2px 0 0 8px',
+  [SpinnerSize.LARGE]: '4px 0 0',
+};
+
+export const getClassNames = memoizeFunction(
+  (color: SpinnerColor, size: SpinnerSize, isCentered: boolean): SpinnerClassNames => {
+    const theme = getTheme();
+    const circleColor = color === SpinnerColor.DARK ? theme.palette.white : theme.semanticColors.link;
+
+    return mergeStyleSets({
+      root: {
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: size === SpinnerSize.LARGE ? 'column' : 'row',
+        justifyContent: isCentered ? 'center' : undefined,
+      },
+      circle: {
```

Comment:
Rather than setting a class on the fabric `Spinner`, did you consider passing a `styles` prop?  Looks like you can style the circle this way:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Spinner/Spinner.styles.tsx#L19

Reply:
oh nice, i'll look at that

Reply:
done, good call